src/main/java/pokemanager/App.java

```diff
@@ -1,71 +1,96 @@
 package pokemanager;
 
 import java.io.*;
+import java.util.Arrays;
+import java.util.List;
 
 public class App {
     private BufferedReader reader;
-    private PrintStream out;
+    public PrintStream pw;
     private Box box;
     private String storageFilename;
+    private List<Command> commands;
+    private boolean run;
 
-    public App(InputStream in, PrintStream out, Box box, String storageFilename) {
+    public App(InputStream in,
+               PrintStream pw,
+               Box box,
+               String storageFilename) {
         this.reader = new BufferedReader(new InputStreamReader(in));
-        this.out = out;
+        this.pw = pw;
         this.box = box;
         this.storageFilename = storageFilename;
+        this.commands = buildCommands();
+        this.run = false;
     }
 
-    public void acceptInput() {
+    private static List<Command> buildCommands() {
+        return Arrays.asList(
+                new RetrieveCommand(),
+                new StoreCommand(),
+                new SaveCommand(),
+                new ExitCommand());
+    }
+
+    private Command findCommand(String command) {
+        for (Command c : commands) {
+            if (c.respondsTo(command)) {
+                return c;
+            }
+        }
+        return null;
+    }
+
+	public String getStoragePath() {
```

Comment:
Is there something strange going on with the indentation or is it my eyes? Shouldn't the `getStoragePath` method be indented the same as `findCommand`?